client/src/components/Burger/Burger.css

```diff
@@ -0,0 +1,42 @@
+.burger-line-one, .burger-line-two, .burger-line-three {
+    height: 8px;
+    width: 50px;
+    background-color: var(--black);
+    margin: 8px 0;
+    transition: .2s ease-in-out;
+    -webkit-transition: .1s ease-in-out;
+    -moz-transition: .2s ease-in-out;
+    -ms-transition: .2s ease-in-out;
+    -o-transition: .2s ease-in-out;
+}
+
+.fade {
+    background-color: rgba(0,0,0,0)
+}
+
+
+.burger-line-one-x {
+    transform: translate(0, 16px) rotate(45deg);
+    -webkit-transform: translate(0, 16px) rotate(45deg);
+    -moz-transform: translate(0, 16px) rotate(45deg);
+    -ms-transform: translate(0, 16px) rotate(45deg);
+    -o-transform: translate(0, 16px) rotate(45deg);
+    transition: .1s ease-in-out;
+    -webkit-transition: .1s ease-in-out;
+    -moz-transition: .1s ease-in-out;
+    -ms-transition: .1s ease-in-out;
+    -o-transition: .1s ease-in-out;
+}
+
+.burger-line-three-x {
+    transform: translate(0, -16px) rotate(-45deg);
+    -webkit-transform: translate(0, -16px) rotate(-45deg);
+    -moz-transform: translate(0, -16px) rotate(-45deg);
+    -ms-transform: translate(0, -16px) rotate(-45deg);
+    -o-transform: translate(0, -16px) rotate(-45deg);
+    transition:  .1s ease-in-out;
+    -webkit-transition:  .1s ease-in-out;
+    -moz-transition:  .1s ease-in-out;
+    -ms-transition:  .1s ease-in-out;
+    -o-transition:  .1s ease-in-out;
+}
```

Comment:
Rather than name these `burger-line-one` and `burger-line-three` can you use a pseudoclass? (i.e. `.burger-line:first-of-type`)

Reply:
This has been updated per the new naming convention.